kubernetes/cache/cache.go

```diff
@@ -136,11 +146,15 @@ func (c *kialiCacheImpl) createCache(namespace string) bool {
 	c.createIstioInformers(namespace, &informer)
 	c.nsCache[namespace] = informer
 
+	if nsChan, exist := c.stopChan[namespace]; !exist || nsChan == nil {
```

Comment:
I think that if the entry has been removed correctly using the delete() invokation, there is no need to check if nsChan is nil here